client-react/src/pages/app/change-app-plan/ChangeAppPlan.tsx

```diff
@@ -146,6 +112,128 @@ export const ChangeAppPlan: React.SFC<ChangeAppPlanProps> = props => {
 
   const subscriptionId = new ArmPlanDescriptor(currentServerFarm.id).subscription;
 
+  const getChangePlanHeaderCard = () => {
+    return (
+      <header>
+        <FeatureDescriptionCard name={t('changePlanName')} description={t('changePlanDescription')} Svg={AppServicePlanSvg} />
+      </header>
+    );
+  };
+
+  const getCurrentPlanDetails = () => {
+    return (
+      <>
+        <Stack style={sectionStyle}>
+          <h4 className={labelSectionStyle}>{t('changePlanCurrentPlanDetails')}</h4>
+        </Stack>
+
+        <ReactiveFormControl id="currentAppServicePlan" label={t('appServicePlan')}>
+          <div tabIndex={0} aria-label={t('appServicePlan') + getPlanName(currentServerFarm)}>
```

Comment:
Insert a space between the label and the app service plan name to avoid screen reader mispronunciation.

```suggestion
          <div tabIndex={0} aria-label={`${t('appServicePlan')} ${getPlanName(currentServerFarm)}`}>
```